test/e2e/load.go

```diff
@@ -117,20 +120,28 @@ var _ = framework.KubeDescribe("Load capacity", func() {
 		podsPerNode int
 		image       string
 		command     []string
+		// What kind of resource we want to create
+		kind schema.GroupKind
 	}
 
 	loadTests := []Load{
 		// The container will consume 1 cpu and 512mb of memory.
-		{podsPerNode: 3, image: "jess/stress", command: []string{"stress", "-c", "1", "-m", "2"}},
-		{podsPerNode: 30, image: "gcr.io/google_containers/serve_hostname:v1.4"},
+		{podsPerNode: 3, image: "jess/stress", command: []string{"stress", "-c", "1", "-m", "2"}, kind: api.Kind("ReplicationController")},
+		{podsPerNode: 30, image: "gcr.io/google_containers/serve_hostname:v1.4", kind: api.Kind("ReplicationController")},
 	}
 
 	for _, testArg := range loadTests {
 		feature := "ManualPerformance"
-		if testArg.podsPerNode == 30 {
+		if testArg.kind.Empty() {
+			testArg.kind = api.Kind("ReplicationController")
+		}
+		if testArg.kind != extensions.Kind("Deployment") && testArg.kind != extensions.Kind("ReplicaSet") && testArg.kind != api.Kind("ReplicationController") {
```

Comment:
Let's remove this if - it shouldn't be needed to check if test is written correctly :)